### PR TITLE
ENG-1444: applying changes agreed in the review meeting

### PR DESCRIPTION
**Pull request by gReis89**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **greis89**

If they are not already assigned, you can assign the PR to them by writing `/assign @greis89` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/ui/component-repository/components/list/ComponentList.js
Comment (alepintus) [inline src/ui/component-repository/components/list/ComponentList.js:49]: Hey @gReis89 ,
Could you please remove this console.log if not necessary?
Thanks

**Review by alepintus — CHANGES_REQUESTED** [review]

File: src/ui/component-repository/components/list/ComponentList.js
Comment (gReis89) [reply src/ui/component-repository/components/list/ComponentList.js]: done, thanks!

**Review by gidesan — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/app-builder/pull/917#pullrequestreview-531301255" title="Approved">gidesan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)~~ [gidesan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
